script.js

```diff
@@ -36,10 +35,6 @@ function addBookToLibrary(book) {
   
```

Comment:
I suggest you to put all variables you defined as global on top of the file, that way it would be easier to see them and more organized(check examples on line 10,11,12,13, 29, etc. )

Reply:
Done